updates/35/updates.sql

```diff
@@ -127,4 +127,94 @@ INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x1495bc9e44Af
+*
+INSERT INTO assets(identifier, name, type) VALUES('AVA', 'Ava AI', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('AVA', 'AVA', 'ava-ai', '', NULL, 1731523296, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x2d5F3B0722aCd35fbb749cB936dfdd93247BBC95', 'BabyBNB', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x2d5F3B0722aCd35fbb749cB936dfdd93247BBC95', 'A', 56, '0x2d5F3B0722aCd35fbb749cB936dfdd93247BBC95', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x2d5F3B0722aCd35fbb749cB936dfdd93247BBC95', 'BABYBNB', 'baby-bnb', '', NULL, 1727458229, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('BABYSHARK', 'Baby Shark Meme', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('BABYSHARK', 'BABYSHARK', 'baby-shark-meme', '', NULL, 1735874640, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0xA91ac63D040dEB1b7A5E4d4134aD23eb0ba07e14', 'Bella', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0xA91ac63D040dEB1b7A5E4d4134aD23eb0ba07e14', 'A', 1, '0xA91ac63D040dEB1b7A5E4d4134aD23eb0ba07e14', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0xA91ac63D040dEB1b7A5E4d4134aD23eb0ba07e14', 'BEL', 'bella-protocol', 'BEL', NULL, 1597392058, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x8443f091997f06a61670B735ED92734F5628692F', 'Bella Protocol', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x8443f091997f06a61670B735ED92734F5628692F', 'A', 56, '0x8443f091997f06a61670B735ED92734F5628692F', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x8443f091997f06a61670B735ED92734F5628692F', 'BEL', 'bella-protocol', 'BEL', NULL, 1601440951, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('BERA', 'Berachain', 'B'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('BERA', 'BERA', 'berachain-bera', 'BERA', NULL, 1737382451, NULL);
+*
```

Comment:
are there erc20 versions of this one in other chains?

Reply:
not that i could find. checked coingecko.